recipe/meta.yaml

```diff
@@ -9,27 +9,28 @@ source:
   sha256: d79f64d14d29dab589a66aa90bd1f7555207bb665829470dba6516c72ebce738
 
 build:
-  number: 0
+  number: 1
   skip: True  # [osx or win or py2k]
   script: python -m pip install --no-deps --ignore-installed .
 
 requirements:
   build:
+    - {{ compiler('fortran') }}
```

Comment:
This is failing miserably when Fortran extensions are present.
@isuruf do you have an idea of what is missing?

Reply:
It's the numpy distutils feature that replaces LDFLAGS. It's rather annoying. Shall we patch numpy for this?

Reply:
Thanks for the explanation @isuruf! I merged the patch and I'll restart things here once the new `numpy` is available.